modules/custom/az_person/config/install/views.view.az_person.yml

```diff
@@ -30,60 +30,12 @@ base_table: node_field_data
 base_field: nid
 display:
   default:
-    display_plugin: default
     id: default
     display_title: Master
+    display_plugin: default
     position: 0
     display_options:
-      access:
```

Comment:
There's lots of changes to Master and other displays. I'm not sure if the changes matter since the displays visually look the same to me.